Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h

```diff
@@ -55,6 +55,12 @@ struct ScrollbarEnabledState {
     bool verticalScrollbarIsEnabled { false };
 };
 
+struct ScrollbarColorState {
+    std::optional<Color> thumbColor;
+    std::optional<Color> trackColor;
+    friend bool operator==(const ScrollbarColorState&, const ScrollbarColorState&) = default;
+};
```

Comment:
Could we make it this and use it as `std::optional<ScrollbarColor>` given we always pass all or no colors?
```suggestion
struct ScrollbarColor {
    Color thumbColor;
    Color trackColor;
    friend bool operator==(const ScrollbarColor&, const ScrollbarColor&) = default;
};
```

I'd even move the struct to `Source/WebCore/platform/` so we can use that struct instead of `std::pair`

Reply:
Fixed.